Rakefile

```diff
@@ -64,7 +69,7 @@ namespace :spec do
         RSpec::Core::RakeTask.new(target.to_sym) do |t|
           ENV['DOCKER_CONTAINER'] = container_name
           t.ruby_opts = '-I ./spec/integration'
-          t.pattern = "spec/integration/*_spec.rb"
+          t.pattern = "spec/integration/[default|docker]_spec.rb"
```

Comment:
Currently, it is unnecessary, but I think it is necessary in the near future.
Because I'll add a test case(s) for #277 , and serverspec file. Probably the new serverspec file is needed only by the new test. Test for `itamae docker` should not load the new serverspec file, so it's necessary.